### PR TITLE
added documentation module

### PR DESCRIPTION
**Pull request by ricklarsen**
Adds content for eventual documentation module / site for Registry module

### CODE REVIEW

File: docs/antora.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: Registry 
+version: 2.21.0-SNAPSHOT
```
Comment (shaundmorris) [inline docs/antora.yml]: ❔ I released Registry 2.22.0 just to get the next version out there. I currently have master at 2.22.1-SNAPSHOT. Thoughts?

File: docs/local-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+site:
+  title: DDF Registry 
+  start_page: home::registry-intro.adoc
+  url: file:///Users/rlarsen/ddf-registry/ddf-registry/docs/build/site/
```
Comment (shaundmorris) [inline docs/local-playbook.yml]: ❔ should this URL be here?

File: docs/pom.xml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.codice.ddf.registry</groupId>
+        <artifactId>registry</artifactId>
+        <version>2.21.0-SNAPSHOT</version>
```
Comment (shaundmorris) [inline docs/pom.xml]: ✏️ just a reminder to double check the latest SNAPSHOT version

**Review by shaundmorris — APPROVED** [review]
❓ Should there be more in the README.md file? Also I assume the `supplemental_ui` files are needed but since I'm not sure figured I would bring it up.

File: docs/antora.yml (hunk 1 above)
Comment (ricklarsen) [reply docs/antora.yml]: I haven't yet figured out the best way to keep them in sync, but I'm looking into it

File: docs/local-playbook.yml (hunk 2 above)
Comment (ricklarsen) [reply docs/local-playbook.yml]: no, i did not mean to commit this playbook yet

File: docs/pom.xml (hunk 3 above)
Comment (ricklarsen) [reply docs/pom.xml]: 👍

**Comment by ricklarsen** [comment]
build now

File: docs/pom.xml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    <parent>
+        <groupId>org.codice.ddf.registry</groupId>
+        <artifactId>registry</artifactId>
+        <version>2.22.1-SNAPSHOT</version>
+    </parent>
+    <groupId>org.codice.ddf.registry</groupId>
+    <artifactId>docs</artifactId>
+    <name>DDF Registry DOCS</name>
+    <properties>
+        <ddf.version>${project.version}</ddf.version>
```
Comment (emmberk) [inline docs/pom.xml]: ✏️ Is this right? Can this line be removed?

**Review by emmberk — APPROVED** [review]
@ricklarsen said that this was tested in a downstream project. LGTM :+1:

File: docs/pom.xml (hunk 4 above)
Comment (ricklarsen) [reply docs/pom.xml]: yes. removed

**Pull request merged**